### PR TITLE
[Release-v2] CE pipeline migration

### PR DESCRIPTION
**Pull request by xabxx**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by xabxx** [comment]
@timrobotson this one is ready.

File: .travis.yml [hunk 1]
```diff
@@ -11,18 +11,19 @@ notifications:
       - travis-build@platform-notifications.robomaker.aws.a2z.com
 env:
   global:
-    - UPSTREAM_WORKSPACE=file
-    - ROSINSTALL_FILENAME=.rosinstall.master
+    - GITHUB_ACCOUNT=aws-robotics
+  # uncomment PACKAGE_NAME to build unit tests - REQUIRED if you'd like to run unit tests using colcon
```
Comment (timrobotson) [inline .travis.yml]: Missing `PACKAGE_NAME` line or `NO_TESTS`, it should have one or the other.

Comment (timrobotson) [reply .travis.yml]: Abby explained how it works. `NO_TESTS` skips all tests, passing `PACKAGE_NAME` builds the package with cmake-target of tests (see https://github.com/aws-robotics/travis-scripts/blob/master/ros1_build.sh#L35), so when the package has tests but doesn't need to be built explicitly with them it's ok to omit both.

Comment (xabxx) [reply .travis.yml]: As discussed offline. This package doesn't need `PACKAGE_NAME` or `NO_TESTS`.

**Pull request merged**